src/content/smart-bulletins/037.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Smart Bulletin No. 037"
+intro: "TikTok Ban; Micro-Purchasing Implications"
+number: "No. 037"
+order: 037
+businessline: "Purchase, Integrated"
+published: "Jun 22, 2023"
+lastupdated: "N/A"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| June 22, 2023 | Implementation of Smart Bulletin #37 |
+
+## Business Line(s) Affected
+
+Purchase, Integrated
+
+## Introduction
+
+This Smart Bulletin is effective upon issuance and provides operational guidance primarily
+focused on micro-purchasing in relation to the requirements of [Public Law 117-328](https://www.congress.gov/bill/117th-congress/house-bill/2617/text)
+Division R, the “No TikTok on Government Devices Act” of the Consolidated Appropriations
+Act of 2023, [OMB Memo M-23-13[PDF, 4 pages]](https://www.whitehouse.gov/wp-content/uploads/2023/02/M-23-13-No-TikTok-on-Government-Devices-Implementation-Guidance_final.pdf), “No TikTok on Government Devices Implementation
+Guidance,” and Federal Acquisition Circular (FAC) 2023-04, Federal Acquisition
+Regulation (FAR) Case 2023-010, subject: “Prohibition on a ByteDance Covered
+Application.”
+
+Public Law 117-328 at Division R (“the Act”) requires the Director of the Office of
+Management and Budget in consultation with the Administrator of General Services, the
+Director of the Cybersecurity and Infrastructure Security Agency, the Director of National
+Intelligence, and the Secretary of Defense to develop standards and guidelines for
+agencies which requires the removal of TikTok software installed on Federal Information
+Technology (IT). TikTok is owned and operated by ByteDance Limited, a Chinese
+company.
+
+OMB Memo M-23-13 mainly addresses requirements for agencies to identify use of
```

Comment:
OMB Memo M-23-13 mainly addresses requirements for agencies to identify the use of